js/renovation/widget.tsx

```diff
@@ -126,6 +129,10 @@ export const viewModelFunction = ({
 };
 
 export const viewFunction = (viewModel: any) => {
+    useLayoutEffect(() => {
+        !viewModel.disabled && viewModel.onContentReady?.();
+    }, [viewModel.children]);
```

Comment:
- why do we use `children` as a dependency?
- why do we use `!viewModel.disabled` here?

Reply:
* `children` is content. I've made it to avoid call `onContentReady` on state changes
* `!viewModel.disabled` - you're right, I'll remove this